run.go

```diff
@@ -253,22 +253,22 @@ func SetCursorShape(shape CursorShapeType) {
 
 // IsFullscreen reports whether the current mode is fullscreen or not.
 //
-// IsFullscreen always returns false on browsers or mobiles.
+// IsFullscreen always returns false on mobiles.
 //
 // IsFullscreen is concurrent-safe.
 func IsFullscreen() bool {
 	return uiDriver().IsFullscreen()
 }
 
-// SetFullscreen changes the current mode to fullscreen or not on desktops.
+// SetFullscreen changes the current mode to fullscreen or not on desktops and browsers.
 //
 // In fullscreen mode, the game screen is automatically enlarged
 // to fit with the monitor. The current scale value is ignored.
 //
 // On desktops, Ebiten uses 'windowed' fullscreen mode, which doesn't change
 // your monitor's resolution.
 //
```

Comment:
Let's add an explanation that fullscreening on browsers require user gestures, and the behavior depends on browser implementations. (Probably this works in Safari in a more limited situation than Chrome, unfortunately)

Reply:
How about
```go
// On browsers, triggering fullscreen requires a user gesture otherwise an error is thrown.
// This behaviour varies across browser implementations.
```
Lmk if this sounds good and I'll commit it.

Reply:
> an error is thrown

This is a little ambiguous. Just `otherwise SetFullscreen does nothing but leave an error message on console`?